source/config/profileUpgradeSteps.py

```diff
@@ -99,3 +99,16 @@ def upgradeConfigFrom_5_to_6(profile: dict):
 	if useInMSWord:
 		from . import AllowUiaInMSWord
 		profile['UIA']['allowInMSWord'] = AllowUiaInMSWord.ALWAYS.value
+
+
+def upgradeConfigFrom_6_to_7(profile: dict):
+	"""
+	Selective UIA registration check box has been replaced with event registration multi choice.
+	"""

```

Comment:
What does `profile['UIA']['eventRegistration']` become as the result of upgrading when:
- the user has not changed the default value of `profile['UIA']['selectiveEventRegistration']` (e.g. it is set to False)
- the user has explicitly set the value to False

I currently only understand the case where:
- the user has explicitly set the value to True

Could you add a summary of what happens for the 3 cases to the docstring?